app/src/main/java/org/mozilla/focus/fragment/InfoFragment.java

```diff
@@ -50,10 +57,24 @@ public View inflateLayout(LayoutInflater inflater, @Nullable ViewGroup container
             webView.setVisibility(View.INVISIBLE);
         }
 
+        applyLocale();
```

Comment:
I moved the applyLocale method code into WebFragment, but I don't think any other fragments that currently extend WebFragment or LocaleAwareFragment would need the same update. The method is abstract within LocaleAwareFragment since different fragments updates differently when the locale is changed.